librepo/python/metadatatarget-py.c

```diff
@@ -81,7 +79,7 @@ check_MetadataTargetStatus(const _MetadataTargetObject *self)
 static int
 metadatatarget_progress_callback(void *data, double total_to_download, double now_downloaded)
 {
-    int ret = LR_CB_OK; // Assume everything will be ok
+    int ret = LR_CB_OK;
```

Comment:
Why are comments being deleted?

Reply:
You know what the line does. It doesn't need a comment.